activerecord/test/cases/adapters/mysql2/unsigned_type_test.rb

```diff
@@ -48,7 +49,6 @@ class UnsignedType < ActiveRecord::Base
       t.unsigned_bigint  :unsigned_bigint_t
       t.unsigned_float   :unsigned_float_t
       t.unsigned_decimal :unsigned_decimal_t, precision: 10, scale: 2
-      t.column :unsigned_zerofill, "int unsigned zerofill"
```

Comment:
This test case name is `test "schema definition can use unsigned as the type"`.
This tests that `unsigned_integer`, `unsigned_bigint`, `unsigned_float`, and `unsigned_decimal` short hand methods creates a column as unsigned.
This line should be in the setup block.